netsim/outputs/ansible.py

```diff
@@ -81,7 +81,7 @@ def get_host_addresses(topology: Box) -> Box:
     if 'loopback' in node:                                  # Create a list of all usable interfaces
       intf_list = [ node.loopback ] + node.interfaces       # ... starting with loopback
```

Comment:
Would be a bit of work to check the templates, but we could consider to include node.loopback in node.interfaces (just like secondary loopbacks)